config/application.yml.default

```diff
@@ -194,7 +194,6 @@ in_person_outage_message_enabled: false
 in_person_proofing_enabled: false
 in_person_proofing_enforce_tmx: false
 in_person_proofing_opt_in_enabled: false
-in_person_proofing_post_office_closed_alert_enabled: true
```

Comment:
After discussing with @eileen-nava we decided to remove the flag while we're cleaning up the other temporary code.